pkg/requests/requests.go

```diff
@@ -3,11 +3,11 @@ package requests
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-
 	"github.com/bitly/go-simplejson"
+	"github.com/pkg/errors"
 	"github.com/pusher/oauth2_proxy/pkg/logger"
+	"io/ioutil"
+	"net/http"
```

Comment:
These imports are now out of order.

Reply:
What import sorting scheme do you use? Goland complains if I swap these around, so I think they're ok according to the gofmt standard.

Reply:
Here's how they look now: 
import (
	"encoding/json"
	"fmt"
	"github.com/bitly/go-simplejson"
	"github.com/pkg/errors"
	"github.com/pusher/oauth2_proxy/pkg/logger"
	"io/ioutil"
	"net/http"
)import (
	"encoding/json"
	"fmt"
	"github.com/bitly/go-simplejson"
	"github.com/pkg/errors"
	"github.com/pusher/oauth2_proxy/pkg/logger"
	"io/ioutil"
	"net/http"
)

Reply:
We use [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) to format and auto include imports in this project.

According to the spec, the import block should be as follows:
```
import (
	"encoding/json"
	"fmt"
	"io/ioutil"
	"net/http"

	"github.com/bitly/go-simplejson"
	"github.com/pkg/errors"
	"github.com/pusher/oauth2_proxy/pkg/logger"
)
```

Third party imports should be after std lib imports and I think this is why CI is currently failing.